taxi/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from django.contrib.auth import get_user_model
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
+        widget=forms.CheckboxSelectMultiple,
+        required=False,
+    )
+
+    class Meta:
+        model = Car
+        fields = "__all__"
```

Comment:
> On the driver list page create a button that leads to the driver creation page.

You must implement the `DriverCreationForm` using `UserCreationForm` as the task requires.

Reply:
I missed that info somehow, bad interpretation